pkg/pool-weighted/contracts/smart/ManagedPool.sol

```diff
@@ -349,6 +363,35 @@ contract ManagedPool is BaseWeightedPool, ReentrancyGuard {
         emit SwapEnabledSet(swapEnabled);
     }
 
+    /**
+     * @dev Getter for the sum of all weights. In initially FixedPoint.ONE, it can be higher or lower
+     * as a result of adds and removes.
+     */
+    function _getDenormWeightSum() internal view returns (uint256) {
+        return _denormWeightSum;
+    }
+
+    /**
+     * @dev Placeholder function for testing. Adding or removing a token may cause the total weight to
+     * diverge from ONE.
+     */
+    function _setDenormWeightSum(uint256[] memory normalizedWeights, uint256 denormWeightSum) internal {
+        InputHelpers.ensureInputLengthMatch(_getTotalTokens(), normalizedWeights.length);
+        _require(denormWeightSum > WeightedMath._MIN_WEIGHT, Errors.MIN_WEIGHT);
```

Comment:
Shouldn't it be:

`uint256 minWeightSum = _getTotalTokens() * WeightedMath._MIN_WEIGHT;`
`_require(denormWeightSum >= minWeightSum, Errors.MIN_WEIGHT); `

Reply:
That would be more accurate, but this is just a placeholder function for testing that will go away, so I wasn't too worried about it. The weight sum will actually change as a result of add/remove operations, and all the real checks would be done in those functions.